README.md

```diff
@@ -2,12 +2,19 @@
 
 ![Build](https://github.com/NHSDigital/prescriptions-for-patients/workflows/Build/badge.svg?branch=master)
 
-This is a specification for the *prescriptions-for-patients* API.
+This is a proxy code and specification for the *prescriptions-for-patients* API.   
+This calls an AWS API that is defined at https://github.com/NHSDigital/prescriptionsforpatients.   
+The API returns prescriptions for the NHS number that the caller is authenticated with.
```

Comment:
Unless otherwise specified in the x-nhs-number header?

Reply:
fixed